src/data/zwave_js.ts

```diff
@@ -83,6 +83,7 @@ export interface ZWaveJSNodeStatus {
   node_id: number;
   ready: boolean;
   status: number;
+  is_secure: boolean | string;
```

Comment:
I thought we know the highest security class they are connected with.

Reply:
We don't get that at the moment at least... Just a book or unknown

Reply:
yes we can replace this with the highest security class once it's available